frontend/components/admin/sidebar.jsx

```diff
@@ -1,15 +1,72 @@
 import React from 'react'
+import Drawer from 'material-ui/Drawer'
+import ActionFlightTakeoff from 'material-ui/svg-icons/action/flight-takeoff';
+import ActionFlightLand from 'material-ui/svg-icons/action/flight-land'
+import MenuItem from 'material-ui/MenuItem'
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import { sectionSelector } from '../../actions/currentSection/currentSection_action'
 
 class AdminSidebar extends React.Component {
   constructor (props) {
     super(props)
+    this.state={
+      showArrowSignIn: false,
+      showArrowSignUp: false
+    }
+  }
+//This is the code for trying to get the arrow to show onClick of a menuItem
+//you're going to run into a error when you click on a MenuItem, just take out this.showArrowSignIn
+  showArrowSignIn(){
+    this.setState({ showArrowSignIn:!this.state.showArrowSignIn });
+  }
+
+  showArrowSignUp(){
+    this.setState({ showArrowSignUp:!this.state.showArrowSignUp })
+  }
+
+  renderArrow(){
```

Comment:
You'll probably want to refactor this now that you're keeping a reference to the selected section name rather than holding booleans for every section